src/fu-util-common.c

```diff
@@ -142,3 +145,48 @@ fu_util_get_user_cache_path (const gchar *fn)
 
 	return g_build_filename (g_get_user_cache_dir (), "fwupd", basename, NULL);
 }
+
+gchar *
+fu_util_get_version (FwupdClient *client, GCancellable *cancellable)
```

Comment:
Hmm, it does look a bit odd hanging the FwupdClient and GCancellable here... maybe do that bit duplicated and then just have `void` here? Sorry to muck you about.